### ISSUE TITLE
Configuring the Plugin - Typo3 Documentation

### ISSUE THREAD

**Issue by jusufcodes** [TYPO3-Documentation/TYPO3CMS-Book-ExtbaseFluid#324]
### Help needed: 

I copied and installed the extension from the documentation after writing the extension step by step failed: https://github.com/TYPO3-Documentation-Examples/store_inventory
See also: https://docs.typo3.org/m/typo3/book-extbasefluid/9.5/en-us/4-FirstExtension/7-configuring-the-plugin.html

As per Documentation I should see an additional extension 'The Store Inventory List'. Which is not shown. I activated the "Store Inventory" Extension already.

Also as illustrated below, the "plugin" tab is missing on my backend (The screenshot is from the documentation).

> After installation of the extension we can insert the plugin on a page. Don’t forget to set the sys folder, in which the products are stored, as the starting point (in our case “Inventory”) in the plugin. Otherwise your products are not found (see figure 4-4).

![image](https://user-images.githubusercontent.com/51062539/67732539-fdb33b00-f9fb-11e9-83ef-8ee76f02d48f.png)

### **What am I missing here?**

The following things I have tried:

+ flush all cache.
+ Deactivate and Reactivate
+ Tried same steps in Typo3 v8.7

System Info:
Typo3 v9.5 & Typo3 v8.7
PHP 7.2 with MariaDB
Local Dev Environment
Extensions managed by Composer

**Comment by DavidBruchmann** [comment]
did you include the extension template in the root-template?

**Comment by jusufcodes** [comment]
Hi David,

thanks for responding. I don't understand how to do that, since in the documentation it is not mentioned.

Do you perhaps suggest the extension should be assigned In the "Backend > Template > Root Page / Root Template" under the Tab Includes? If yes, then I can't assign the extension template.

**Comment by sypets** [comment]
> As per Documentation I should see an additional extension 'The Store Inventory List'. Which is not shown. I activated the "Store Inventory" Extension already.

You mean, you do not see the plugin "The Store Inventory List" listed in the list of plugins in the plugin tab?

> Also as illustrated below, the "plugin" tab is missing on my backend

The plugin tab is actually missing? That seems like a different problem unrelated to the extension.

?

I cannot confirm the missing "Store Inventory List". I did the following:

On existing 8.7 systems (with + without Composer):

1. Install latest store_inventory extension from GitHub:  git@github.com:TYPO3-Documentation-Examples/store_inventory.git
2. Create a new element type "plugin"
3. Switch to plugin tab : I see "Store Inventory List"

It was not necessary to flush the cache.

![store_inventory](https://user-images.githubusercontent.com/13206455/67739044-5e00a780-fa11-11e9-8e19-3e844edee2f0.png)

What might have gone wrong?

* Managing with Composer: The store_inventory does not include a composer.json. How did you do the installation?
* Include static include TypoScript templates is not necessary for that extension (though that is a good tip in most cases). The extension does not include TypoScript as you can see as there is no Configuration/TypoScript folder.

---

About the documentation: We are aware that some of the documentation / example extension is currently outdated. We are working on this as fast as possible.

You can also get support for TYPO3 by the ways listed on the https://typo3.org/help page.

**Comment by jusufcodes** [comment]
Hey sypets,

thanks for responding, your comment helped a lot. Took me a while, but I managed to display the Product List finally (in Typo3 v9.5).

> As per Documentation I should see an additional extension 'The Store Inventory List'. Which is not shown. I activated the "Store Inventory" Extension already.

>> You mean, you do not see the plugin "The Store Inventory List" listed in the list of plugins in the plugin tab?

Yes, but I was wrong. I misunderstood extension and plugin for the same. Also the following sentence:
> "After installation of the extension we can insert the plugin on a page." 

This sentence made me think the page itself should be edited. Instead I have to create a content in a page and select the plugin inside of the content. Makes sense.

> Managing with Composer: The store_inventory does not include a composer.json. How did you do the installation?

I just added the extension to the typo3conf/ext path. It allows the extension to be activated. But quickly discovered that composer.json is mandatory for the autoload. 
I have configured my composer to allow local packages to be required.

Greatly appreciate the help. I'll close the issue now.
